docs/developer-guide/how-to-change-background-list.md

```diff
@@ -0,0 +1,39 @@
+# How to change the background list
+
+In order to edit the background list of the new map configuration you have to:
+
+- open **new.json** file
+- edit the **layers** property
+
+You can add new background layer by adding a new entry in the layers list (array).
+
+The important thing is the layer group must be **"background"** and only one *background* layer must have visibility set to `true`
+
+[Here](../maps-configuration/#layers-options) you can find more info on how a layer can be configured.
+
+```
+"layers": [
+    {this can be another background layer or a layer that is stored in TOC},
+    {same as above},
+	{
+	    "type": "osm",
+		"title": "Open Street Map",
+		"name": "mapnik",
+        "source": "osm",
+		"group": "background", <--- see here
```

Comment:
This code will be copied and pasted, so maybe it is not a good idea to put <--- comments there

Reply:
right